aurpublish.in

```diff
@@ -147,6 +147,18 @@ fi
 if [[ "$url_set" != "true" ]]; then
     # use git configured remoteUrl, if set
     REMOTE_URL=$(git config --default "${REMOTE_URL}" --get aurpublish.remoteUrl | sed 's:/*$::')
+
+    # warn the user if they have a non-default ssh alias for 'aur'
+    auralias=$(ssh -G aur | awk '$1 == "hostname" { print $2 }')
+    if [[ "$auralias" != 'aur' && "$auralias" != 'aur.archlinux.org' ]]; then
```

Comment:
Existing style would suggest ${auralias}.

n.b. quoting inside of `[[ ]]` is unnecessary (bash does not perform word splitting or pathname expansion inside the double brackets version).

Reply:
Fixed!

Reply:
Note that below with $userwarned there is also the brackets-style inconsistency. :)

Reply:
Double fixed!